content/hardware/10.mega/boards/giga-r1-wifi/tutorials/giga-camera/giga-camera.md

```diff
@@ -16,38 +16,46 @@ The GIGA R1 has a dedicated camera connector that allows certain camera modules
 In this guide, we will explore the following:
 
 - Where the camera connector is located.
-- What cameras are compatible.
-- What library to use.
-- How to setup a camera stream to a processing application.
+- What cameras are compatible?
+- What library to use?
+- How to set up a camera stream to a browser using webserial.
 
 ## Hardware & Software Needed
 
 To follow and use the examples provided in this guide, you will need an [Arduino GIGA R1 WiFi](/hardware/giga-r1-wifi)
 
 You will also need the following software:
+
 - [Arduino IDE](https://www.arduino.cc/en/software) (any version).
-- [Processing](https://processing.org/download) (for displaying camera feed).
+- [Webserial](https://labs.oniudra.cc/en/labs/web-serial-camera) (for displaying camera feed).
+
+To run the Webserial locally go to:
+
+```
+C:\Users\<UserName>\AppData\Local\Arduino15\packages\arduino\hardware\mbed_giga\4.1.3\libraries\Camera\extras\WebSerialCamera
+```
+
```

Comment:
Not sure if we even want to talk about this option. If we do we need to put the paths for all 3 operating systems.

Reply:
Good point. I removed it for now and in case we get any requests about it we can add it for all each OS.